src/main/java/io/druid/data/input/impl/JSONParseSpec.java

```diff
@@ -19,26 +19,46 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser.Feature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.metamx.common.parsers.JSONParser;
 import com.metamx.common.parsers.Parser;
 
+import java.util.LinkedHashMap;
 import java.util.List;
+import java.util.Map;
 
 /**
  */
 public class JSONParseSpec extends ParseSpec
 {
   private final ObjectMapper objectMapper;
+  private final LinkedHashMap<String, Boolean> featureSpecs;
 
   @JsonCreator
   public JSONParseSpec(
       @JsonProperty("timestampSpec") TimestampSpec timestampSpec,
-      @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec
+      @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec,
+      @JsonProperty("featureSpec") LinkedHashMap<String, Boolean> featureSpecs
   )
   {
     super(timestampSpec, dimensionsSpec);
+    this.featureSpecs = (featureSpecs == null) ? new LinkedHashMap<String, Boolean>() : featureSpecs;
     this.objectMapper = new ObjectMapper();
+    for (Map.Entry<String, Boolean> entry : this.featureSpecs.entrySet()) {
+      try {
+        Feature feature = Feature.valueOf(entry.getKey());
+        objectMapper.configure(feature, entry.getValue());
+      }
+      catch (Exception e) {
```

Comment:
Any reason that the exception is suppressed?

Actually throwing an exception might be nice, so users know quickly if they spelled a feature wrong or if they are trying to set an unsupported feature.